### DISCUSSION TITLE
Brand new text editor for the operators. LHC 4.52, where activate?

### DISCUSSION THREAD

**Discussion by PeopleInside** [LiveHelperChat/livehelperchat#2168] [Q&A] [answered]
Hi, I hope the day and the week is going well 🌞🍀🌻
Thank you very much for the wonderful Live Helper Chat software!

I saw that in the LHC version [4.52](https://github.com/LiveHelperChat/livehelperchat/releases/tag/4.52v) a new text editor for the operator has been released. The release notes say that the new editor can be activated from the Dashboard but where exactly?

I was looking for the right path to activate this new editor and see how it looks like but I was unable to see, to find it.
Thank you ❤️.

**Comment by remdex** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/2538a2bc-8e95-4dc7-98b5-6cf68842c43b)

**Reply by PeopleInside** [reply]
Thank you 😁
